packages/model-registry/upstream/frontend/src/app/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionDetails.tsx

```diff
@@ -142,4 +145,19 @@ const ModelVersionsDetails: React.FC<ModelVersionsDetailProps> = ({ tab, ...page
   );
 };
 
+const ModelVersionsDetails: React.FC<ModelVersionsDetailProps> = (props) => {
+  const { preferredModelRegistry } = React.useContext(ModelRegistrySelectorContext);
+  const { modelVersionId: mvId } = useParams();
+  
+  const labelSelectors = React.useMemo(() => {
+    return mvId ? { [KnownLabels.MODEL_VERSION_ID]: mvId } : undefined;
+  }, [mvId]);
```

Comment:
~~I think the labelSelectors have to always also look for the label with the model registry name, to prevent showing the wrong thing if the user has multiple registries and deployments from the same model/version ids that exist in another registry. `[KnownLabels.MODEL_REGISTRY_NAME]: preferredModelRegistry`. We need to make sure that's included everywhere we are using MRDeploymentsContextProvider.~~

Huh, interesting... this isn't necessary because that filter gets applied all the way down in the kserve package. [Slack thread about it](https://redhat-internal.slack.com/archives/C07BFD5J4CB/p1756928453068389). tech debt for another day...